### PR TITLE
Add screen to append to kernel cmdline

### PR DESCRIPTION
**Pull request by matthewrsj**
Allow the user to append parameters to the kernel cmdline. This screen
shows an example of the sane defaults used in Clear Linux for native
kernel. The appended parameters are added to <target>/etc/kernel/cmdline
and clr-boot-manager update is run to append the parameters to the boot
entry.

Signed-off-by: Matthew Johnson <matthew.johnson@intel.com>

### CODE REVIEW

**Comment by matthewrsj** [comment]
Eh, a pylint fix for the nvme test made it into this commit. I can take it out and add it in another commit if you wish.

**Comment by bryteise** [comment]
No need, can't really tell ;).

**Comment by matthewrsj** [comment]
Forgot to update expect tests again. Hold on.

File: ister_gui.py [hunk 1]
```diff
@@ -2880,33 +2930,48 @@ def __init__(self, *args, **kwargs):
...
+        config_hostname = ConfigureHostname(next(current), total)
+        # User configuration meta-step
+        step = next(current)
+        confirm_user = ConfirmUserMenu(step, total)
+        user_configuration = UserConfigurationStep(step, total)
+        bundle_selector = BundleSelectorStep(next(current), total)
+        # Networking meta-step
+        step = next(current)
+        confirm_dhcp = ConfirmDHCPMenu(step, total)
+        static_ip_config = StaticIpStep(step, total)
```
Comment (bryteise) [inline ister_gui.py]: I like the use of the counter but it gets a little confusing to me when the code alternates between next(current) and step when setting up the orderings. Maybe some reordering or more descriptive variable names as I see this being a possible issue for somebody who wants to change the list order in the future figuring out where to insert what potentially.

Comment (matthewrsj) [reply ister_gui.py]: I have to save next(current) to a variable if I want to use the int more than once before it is incremented.

I'll change the variable names and make the comments more verbose.

Comment (bryteise) [reply ister_gui.py]: Thanks

**Comment by matthewrsj** [comment]
@bryteise expect tests updated

**Pull request merged**